Source/cmNinjaNormalTargetGenerator.cxx

```diff
@@ -546,10 +560,12 @@ std::vector<std::string> cmNinjaNormalTargetGenerator::ComputeDeviceLinkCmd()
   return linkCmds;
 }
 
-std::vector<std::string> cmNinjaNormalTargetGenerator::ComputeLinkCmd(
-  const std::string& config)
+std::pair<std::vector<std::string>, std::vector<std::string>>
+cmNinjaNormalTargetGenerator::ComputeLinkCmd(const std::string& config,
+                                             std::set<int>& ranlibIndex)
 {
   std::vector<std::string> linkCmds;
+  std::vector<std::string> commands;
```

Comment:
Please rename this variable so it is clear what are the commands, for example: "filteredCommands"